src/main/java/org/pokecollect/service/PokecardServiceStub.java

```diff
@@ -10,22 +10,16 @@
 @Service
 public class PokecardServiceStub implements IPokecardService {
 
-    @Autowired
-    public IPokecardDAO pokecardDAO;
-
-    public PokecardServiceStub(){
-
-    }
+    private final IPokecardDAO pokecardDAO;
```

Comment:
Switched to constructor injection for better testability and to ensure immutability